cms/static/js/views/import.js

```diff
@@ -97,8 +97,8 @@ define(
             $.cookie(COOKIE_NAME, JSON.stringify({
                 file: file,
                 date: moment().valueOf(),
-                completed: completed || false
-            }));
+                completed: completed || false,
+            }), {path: window.location.pathname});
```

Comment:
I am not sure about the time period but my question is why we choose short period?